config.yml.example

```
@@ -97,3 +97,15 @@ jail:
     link_influxdb: influxdb
     influxdb_password: unifi-poller
     poller_password: upoller
+
+  grafana:
+    blueprint: grafana
+    ip4_addr: 192.168.1.100/24
+    gateway: 192.168.1.1
+    password: grafana
+    link_influxdb: influxdb
```

maybe change this to be clear the value is a jailname:
```suggestion
    link_influxdb: influxdbjail
```

---

I think this should be standard for all jails configs. If we want to change it then it's easy, but I think it should be the same for all.

---

You are right, could you do this and also change the jailnames in the example accordingly?